core/postfix/requirements.txt

```diff
@@ -0,0 +1,3 @@
+podop==0.2.5
+postfix-mta-sts-resolver==1.1.4
+socrate==0.2.0
```

Comment:
what's this file useful for? it's all in the base image, no?

Reply:
It lists the requirements of this container and is used when building the base image.
I'll remove it and create a single location with all requirements as discussed in element.

Reply:
I've moved all requirements to the base container.

- `requirements-build.txt` contains the pinned build-time dependencies (pip, setuptools, wheel)
- `requirements-dev.txt` contains mailu's direct dependencies (not pinned)
- `requirements-prod.txt` contains all pinned depencies to build the image